.travis.yml

```diff
@@ -33,11 +37,8 @@ before_script:
 
 script:
 - npm install
-- npm run build
-- npm run test
-- npm run test:saucelabs
-- npm run test:report
-- npm run functional
+- npm run test:slreport
+- npm run functional:ci
```

Comment:
The "functional:ci" script builds and unit tests runs during the build step. The script also runs http-serve and the the Testcafe in parallel.